src/actions/formatSelection.ts

```diff
@@ -6,32 +7,34 @@ import suppressFocusStore from '../stores/suppressFocus'
 
 /** Format the browser selection or cursor thought as bold, italic, strikethrough, underline. */
 export const formatSelectionActionCreator =
-  (command: 'bold' | 'italic' | 'strikethrough' | 'underline'): Thunk =>
+  (
+    command: 'bold' | 'italic' | 'strikethrough' | 'underline' | 'foreColor' | 'backColor' | 'removeFormat',
+    color: string = '',
+    colors: any = {},
```

Comment:
Can you explain the difference between `color` and `colors`? Setting a proper type will help with the understanding as well.